### PR TITLE
Feat create graphql client

### PR DESCRIPTION
**Pull request by hugodnpm**
#3 
Added and config urql.
Added .env-example

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+      <AccountProvider account={pageProps.account}>
+        <Component {...pageProps} />
+      </AccountProvider>
+    </Provider>
+  );
 }
 
-export default MyApp
+export default MyApp;
+//  </AccountProvider>
```
Comment (RenatoSiqueira) [inline pages/_app.tsx]: Dead code. Why?

**Review by RenatoSiqueira — CHANGES_REQUESTED** [review]

File: pages/[slug]/index.tsx [hunk 2]
```diff
@@ -1,56 +1,57 @@
...
       <Head>
         <title>{account?.friendlyName}</title>
       </Head>
-      <h1>Welcome {account?.friendlyName}</h1>
+      <h1>Welcome hugo! {account?.friendlyName}</h1>
       <p>
-        <Link href='/about'>
-          <a>About</a>
+        <Link href="/about">
+          <a>hugo-About</a>
```
Comment (RenatoSiqueira) [inline pages/[slug]/index.tsx]: Remove this names, please :D

**Review by RenatoSiqueira — CHANGES_REQUESTED** [review]

**Comment by RenatoSiqueira** [comment]
Please, resolve conflicts above

File: pages/_app.tsx (hunk 1 above)
Comment (hugodnpm) [reply pages/_app.tsx]: I'm sorry, removed

File: pages/[slug]/index.tsx (hunk 2 above)
Comment (hugodnpm) [reply pages/[slug]/index.tsx]: Ok! Removed

**Review by RenatoSiqueira — APPROVED** [review]

**Pull request merged**